index.js

```diff
@@ -85,8 +87,14 @@ const checkAnswer = () => {
             enableButton(next_button)
             if (answerBtn.dataset.winner === 'true' && !buttonClicked){
             answerBtn.style.backgroundColor = "#98d03b"
+            score ++
+            guesses ++
+            document.querySelector('.scoreCounter').textContent = score + "/" + guesses
+            increment5(score)
             } else if (answerBtn.dataset.winner === 'false' && !buttonClicked) {
             answerBtn.style.backgroundColor = "#d94536"
+            guesses ++
+            document.querySelector('.scoreCounter').textContent = score + "/" + guesses
```

Comment:
repeated line of code - perhaps make into a function to be used in the two places eg: pass in score and guesses and have it update the DOM